src/app/components/GoogleSheetsInput.svelte

```diff
@@ -73,10 +73,11 @@
    *
    */
   function getSpreadsheetId() {
-    googleSheetsURL = document.getElementById("googleSheetsURL").value;
+    // TODO: should not use id; should make use of bindings
+    googleSheetsURL = document.getElementById("input-googleSheetsURL").value;
 
     const googleSheetRegExp = RegExp(
-      "http(s?)://docs.google.com/spreadsheets/[a-z]+/[a-zA-z0-9]+/[a-zA-z0-9#=]+"
+      "http(s?)://docs.google.com/spreadsheets/[a-z]{1}/[\\w-]+/[\\w#=-]+"
```

Comment:
Interesting! So the old regex was too restrictive? How did you come up with this regex?

Also, `{1}` is always redundant — it means select the previous token exactly once, which is the default:

```suggestion
      "http(s?)://docs.google.com/spreadsheets/[a-z]/[\\w-]+/[\\w#=-]+"
```

Reply:
I noticed that mine includes underscores and dashes, so I've included them: `https://docs.google.com/spreadsheets/d/11NohN7t_iZ1hRx-0J7Qtr7ZMVlGl0ffjui5HLqzwhmU`. I'd love to take a look if there's documentation that I can refer back to!